users/users/users.php

```diff
@@ -150,6 +150,8 @@ public function post()
 				
 			}
 			*/
+			$profiles = FD::model( 'profiles' );
+			$profiles->updateUserProfile($user->id,$data['profile_id']);
```

Comment:
Need a check to verify if `$data['profile_id']` is a valid profile type id. If this check is not being performed in the updateUserProfile method, you need to implement that check. If the check fails, the user should be kept in the default profile id.